mnelab/dialogs/__init__.py

```diff
@@ -18,13 +18,32 @@
 from .metainfodialog import MetaInfoDialog
 from .montagedialog import MontageDialog
 from .pickchannelsdialog import PickChannelsDialog
+from .plotevokedcomparisondialog import PlotEvokedComparisonDialog
```

Comment:
The module could be called `plotevokeddialog` and contain both dialogs.

Reply:
Good idea!